fictadvisor-api/src/v2/mappers/DisciplineMapper.ts

```diff
@@ -50,7 +51,7 @@ export class DisciplineMapper {
         position: teacher.position as Position,
         rating: +teacher.rating,
         disciplineTeacherId: disciplineTeacher.id,
-        roles: disciplineTeacher.roles.map((r) => r.role),
+        roles: disciplineTeacher.roles.map(({ disciplineType }) => TeacherRoleAdapter[disciplineType.name]),
```

Comment:
looks like some [common logic](https://github.com/fictadvisor/fictadvisor/pull/239/files#diff-6d1d9d2dbead3f16435662f5bbe185bdc158748dece63282194d4ef389eeddcaR134), should we move it to some util or a mapper?